routes/index.js

```diff
@@ -76,6 +76,10 @@ var routes = {
     };
     var request_id = request.headers['x-request-id'];
 
+    if (transaction.description === "Thunderbird") {
+      metadata.thunderbird = true;
+    }
```

Comment:
OK, that works for me. I wonder why we didn't try that in the first place for description?

So this makes me wonder if accounting knows how to get thunderbird subscriptions at all? My guess is no.

Maybe we're not passing description at all for subscriptions and we'll need to pass it into metadata anyway. Thus, we would end up with description and the boolean, which, if you're fine with, so I am?